CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ _This release is scheduled to be released on 2024-07-01._
 
 ### Added
 
+- [compliments] Added `specialDayUnique` config option, defaults to `false`. When set to `true` it replaces the existing compliments array with only the compliments that have been configured for that day (#3465)
```

Comment:
Can be shorter: 

```suggestion
- [compliments] Added `specialDayUnique` config option, defaults to `false`. (#3465)
```

a longer explanation should be added in a seperate PR to the MM docs repo: https://github.com/MagicMirrorOrg/MagicMirror-Documentation